src/todo/logging.lua

```diff
@@ -0,0 +1,21 @@
+-- TODO: String format support missing
```

Comment:
This should be a new issue.

Reply:
Yeah. I just noticed this but yes.. the logging will get some love at some point but not here :)